earth_enterprise/src/fusion/autoingest/AssetFactory_unittest.cpp

```diff
@@ -51,17 +62,96 @@ class MockAssetStorage
               }
 };
 
+
+// added ID field to aid in comparison
```

Comment:
Minor: Not sure this comment needs to remain.

Reply:
I could go either way. That field wasn't there originally and it might make things more clear for anyone later on looking to change the mock-up.